### ISSUE TITLE
Lint warnings

### ISSUE THREAD

**Issue by jaalto** [alpaker/fill-column-indicator#37]
FYI, The following warnings are displayed for the *.el file

```
*** Lint Emacs 24.3.1 2013-09-05 00:36:16 file: fill-column-indicator.el
** Lint M-x lm-verify (list-mnt.el)
Missing: ;; Maintainer:
Missing: ;; URL:
Missing: ;;; History:
** Lint M-x checkdoc-current-buffer (checkdoc.el)
fill-column-indicator.el:221: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:230: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:242: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:252: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:273: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:282: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:304: Lisp symbol `truncate-lines' should appear in quotes
fill-column-indicator.el:315: Lisp symbol `line-move-visual' should appear in quotes
fill-column-indicator.el:409: Argument `all-frames' should appear (as ALL-FRAMES) in the doc string
fill-column-indicator.el:516: Argument `blank' should appear (as BLANK) in the doc string
fill-column-indicator.el:516: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:595: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:603: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:630: Argument `body' should appear (as BODY) in the doc string
fill-column-indicator.el:647: Arguments occur in the doc string out of order
fill-column-indicator.el:651: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:672: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:695: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:743: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:748: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:753: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:770: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:782: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:800: Argument `_ignored' should appear (as _IGNORED) in the doc string
fill-column-indicator.el:800: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:810: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:817: Argument `start' should appear (as START) in the doc string
fill-column-indicator.el:817: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:840: Argument `start' should appear (as START) in the doc string
fill-column-indicator.el:840: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:845: Argument `all-frames' should appear (as ALL-FRAMES) in the doc string
fill-column-indicator.el:845: Lisp symbol `fill-column' should appear in quotes
fill-column-indicator.el:850: Lisp symbol `fill-column' should appear in quotes
```

**Comment by alpaker** [comment]
Just FYI, I'm abandoning this package and welcoming requests to be take over as maintainer.

**Comment by alpaker** [comment]
The functionality provided by this package has now been implemented natively in Emacs as `display-fill-column-indicator-mode`, available as of version 27.0.90. Unless you are forced to use an older Emacs, you should use the native implementation instead of `fci-mode`. The former is unaffected by most of the compatibility issues that this add-on has.
